src/utils/number.tsx

```
@@ -1,3 +1,5 @@
+import { parseInt } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)